### ISSUE TITLE
Formulaic errors in PAML property descriptions

### ISSUE THREAD

**Issue by mszulcz-mitre** [Bioprotocols/PAML-specification#12]
The descriptions of class properties in the PAML Data Model frequently have the same style of error.  Maybe these descriptions were auto-generated?  Here's a few examples:

https://github.com/Bioprotocols/PAML-specification/blob/d86de95f805889adc4d0808b53fefefba79406e6/pamlDataModel.tex#L114-L115

https://github.com/Bioprotocols/PAML-specification/blob/d86de95f805889adc4d0808b53fefefba79406e6/pamlDataModel.tex#L117-L118

https://github.com/Bioprotocols/PAML-specification/blob/d86de95f805889adc4d0808b53fefefba79406e6/pamlDataModel.tex#L331

https://github.com/Bioprotocols/PAML-specification/blob/d86de95f805889adc4d0808b53fefefba79406e6/pamlDataModel.tex#L416-L419

**Comment by danbryce** [comment]
Yes, these files are autogenerated.  We are in the process of updating the standard generation and plan to remove the generated files from the repository.  Thanks!
